source/agora/consensus/state/UTXODB.d

```diff
@@ -73,7 +73,7 @@ package class UTXODB
         scope int delegate (const ref Hash, const ref UTXO) @safe dg) @safe
     {
         return () @trusted {
-            auto results = this.db.execute("SELECT hash, unlock_height, type, amount, locktype, lock FROM utxo");
+            auto results = this.db.execute("SELECT hash, unlock_height, type, amount, locktype, lock FROM utxo order by hash");
```

Comment:
Use uppercase for SQL keywords, as done everywhere else.
Also that's a good idea, could you make it its own PR ?

Reply:
Updated in new PR https://github.com/bosagora/agora/pull/2348